app/Http/Controllers/QuestionController.php

```diff
@@ -153,22 +162,30 @@ public function show(Question $question)
                 'comments.user',
             ])
             ->get();
-        $answers->map(function ($answer) {
+        $answers->map(function ($answer) use ($user_id) {
             $maxContentVersion = $answer->contents()->max('version');
             $answer->content = $answer->contents()
                 ->where('version', $maxContentVersion)
                 ->first();
 
             $answer->sum_votes = $answer->votes->sum('vote');
+
+            if ($user_id > config('constants.zero')) {
```

Comment:
```suggestion
            if ($user_id > 0) {
```
so sánh ntn ko cần cho vào config nhé. khó hiểu thêm

Reply:
Anh Công mấy pull trước comment em chị ạ @@